Scripts/assignment/emme_assignment.py

```diff
@@ -296,16 +298,22 @@ def print_vehicle_kms(self):
         emmebank = self.emme.modeller.emmebank
         freight_classes = ["van", "truck", "trailer_truck"]
         vdfs = [1, 2, 3, 4, 5]
-        transit_modes = ["bde", "g", "m", "rj", "tp"]
+        transit_modes = {
+            "bus": "bde",
+            "trunk": "g",
+            "metro": "m",
+            "train": "rj",
+            "tram": "tp",
+        }
         kms = dict.fromkeys(freight_classes + ["car"])
         for ass_class in kms:
             kms[ass_class] = dict.fromkeys(vdfs)
             for vdf in kms[ass_class]:
                 kms[ass_class][vdf] = 0
-        transit_kms = dict.fromkeys(transit_modes)
+        transit_dists = dict.fromkeys(transit_modes)
         transit_times = dict.fromkeys(transit_modes)
         for mode in transit_modes:
-            transit_kms[mode] = 0
+            transit_dists[mode] = 0
             transit_times[mode] = 0
```

Comment:
**dict.fromkeys** accepts _(2nd optional paraml)_ default value, you can pass 0 -> initializes {key1: 0, key2: 0, etc.}